lib/metadata/metadataUtils.js

```diff
@@ -263,3 +262,10 @@ export function metadataValidateBucket(params, log, callback) {
         return callback(null, bucket);
     });
 }
+
+module.exports = {
+    parseListEntries,
+    metadataGetObject,
+    metadataValidateBucketAndObj,
+    metadataValidateBucket,
+};
```

Comment:
It is, since we don't use them outside of this file. We can export them later if we need to at some point.